recipes/nbexamples/meta.yaml

```diff
@@ -9,7 +9,7 @@ package:
 source:
   fn: {{ gh_repo }}-v{{ version }}.tar.gz
   url: https://github.com/{{ gh_org }}/{{ gh_repo }}/archive/v{{ version }}.tar.gz
-  sha256: 96926c36333e57aee3820722f1107fbf6a3df9113b8585b14e572fccce385923
+  sha256: f391b42ea6856d92a06fafa18ede464e400c9b2804a2a022d0b7197fd4bd4061
```

Comment:
Given this is seeing a wider range of effect, I have [posted]( https://gitter.im/conda-forge/conda-forge.github.io?at=57e86336ea1b6b1a6c68ffb0 ) details on Gitter about this change.